Directory.Build.props

```
@@ -3,15 +3,27 @@
   <PropertyGroup Label="Сommon properties">
     <LangVersion>latest</LangVersion>
     <PackageProjectUrl>https://github.com/graphql-dotnet/server</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/graphql-dotnet/server</RepositoryUrl>
     <RepositoryType>Git</RepositoryType>
```

```suggestion
    <RepositoryType>git</RepositoryType>
```

from here https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/msbuild-targets.md#pack-target